packages/mermaid-layout-elk/src/render.ts

```diff
@@ -3,6 +3,14 @@ import ELK from 'elkjs/lib/elk.bundled.js';
 import type { InternalHelpers, LayoutData, RenderOptions, SVG, SVGGroup } from 'mermaid';
 import { type TreeData, findCommonAncestor } from './find-common-ancestor.js';
 
+type Node = LayoutData['nodes'][0];
+
+interface NodeWithVertex extends Omit<Node, 'domId'> {
+  children?: unknown[];
+  labelData?: any;
```

Comment:
```suggestion
  labelData?: unknown;
```

Reply:
Unfortunately, `unknown` caused type issues.

Instead, I've fully typed `labelData?:` by creating a new `interface LabelData` type: https://github.com/mermaid-js/mermaid/pull/5974/commits/6cc0132e0a192c4101c86083f34ca3af7d6ebe23

To be honest, I don't know what it does (to me, it seems like some fields aren't use anywhere?), but I've typed it with the values it seems to have.